azure-functions/acbs-function/.funcignore

```diff
@@ -0,0 +1,10 @@
+*.js.map
+*.ts
+.git*
+.vscode
+local.settings.json
+test
+getting_started.md
+node_modules/@types/
+node_modules/azure-functions-core-tools/
+node_modules/typescript/
```

Comment:
These don't seem to correspond to any files in my `node_modules`.

If we do want these, it would be better to automatically add dev-dependencies from the `package.json` here